src/OpenSage.Game/Logic/Object/Create/CreateModule.cs

```diff
@@ -2,7 +2,7 @@
 {
     public abstract class CreateModule : BehaviorModule, ICreateModule
     {
-        private bool _shouldCallOnBuildComplete;
+        private bool _shouldCallOnBuildComplete = true;
```

Comment:
@tgjones I think you recently modified the code here so I wanted to check with you to see if this has any adverse effects. Currently it's only used in (and is needed for) `SupplyCenterCreate.OnBuildCompleteImpl()`
https://github.com/OpenSAGE/OpenSAGE/blob/71f6625b3b8ee500b9d7626615fa6792c53799fd/src/OpenSage.Game/Logic/Object/Create/SupplyCenterCreate.cs#L14-L20